### DISCUSSION TITLE
Can collection hold data of any other non-English language

### DISCUSSION THREAD

**Discussion by basavarajdodamani** [strapi/strapi#7421] [Legacy Topics]
Hello,
We are building an application and using STRAPI as UI content management tool. Static text would be stored in one of the collections and we would like to have this data in another language such as Arabic. Could we do that? If possible, Would there be any encoding issues etc? Please let us know the considerations for this usecase.

**Comment by derrickmehaffy** [comment]
You would want to pay attention to the encoding schema being used by the database before you attach Strapi to it. I know in cases like MySQL/MariaDB you can change the encoding for a specific database when you create it. There should be no issues handling the encoding *for data* within Strapi but models and fields need to be in english for the moment.

**Reply by basavarajdodamani** [reply]
Thank you @derrickmehaffy for your response. I am using MongoDB as database, looks like MongoDB enterprise edition only supports Arabic language. Also as i checked, strapi can hold arabic data in a collection and returns the data as is without any encoding issues, which is okay for now.
